conf/application-json-logger.xml

```diff
@@ -1,13 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<configuration>
-
-    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
-        <encoder class="uk.gov.hmrc.play.logging.JsonEncoder"/>
-    </appender>
-
-    <logger name="application.uk.gov.hmrc.onestopshopforexrates.services.ExchangeRatesServiceImpl" level="INFO"/>
```

Comment:
From my understanding of this logger change, we need to add it to a different place, rather than in this file. This logged will need be set to INFO still, I believe it's in application.conf or in app-config-env